cloud/openstack/os_server_facts.py

```diff
@@ -64,10 +77,16 @@ def main():
 
     try:
         cloud = shade.openstack_cloud(**module.params)
-        server = cloud.get_server(module.params['server'])
-        hostvars = dict(openstack=meta.get_hostvars_from_server(
-            cloud, server))
```

Comment:
So this is also changing the default behavior of the module, which is to provide all the details in the facts. The default now would be to just provide a list. Is there a strong reason to change the default?